shared/config/hyperdrive-config.go

```diff
@@ -163,6 +164,20 @@ func newHyperdriveConfigImpl(hdDir string, network config.Network) *HyperdriveCo
 			},
 		},
 
+		EnableIPv6: config.Parameter[bool]{
+			ParameterCommon: &config.ParameterCommon{
+				ID:                 ids.EnableIPv6ID,
+				Name:               "IPv6 Enabled",
```

Comment:
Should be `Enable IPv6` since it's an action they can take; all of the other parameters are named in that fashion.